crypto/fipsmodule/rand/rand.c

```diff
@@ -48,9 +48,22 @@
 // This might be a bit of a leap of faith, esp on Windows, but there's nothing
 // that we can do about it.)
 
+// When in FIPS mode we use the CPU Jitter entropy source to seed our DRBG.  
+// This entropy source is very slow and can incur a cost anywhere between 10-60ms
+// depending on configuration and CPU.  Increasing to 2^24 will amortize the 
+// penalty over more requests.  The value was selected by reviewing OpenSSL and 
```

Comment:
I think it would be good to add here that this is the value used in OpenSSL. Is it used in OpenSSL 3.0?

Reply:
Yes, this is the value used in OpenSSL 3.0.  I'll add a comment.